config/manager/kustomization.yaml

```diff
@@ -1,16 +1,16 @@
 resources:
-  - manager.yaml
+- manager.yaml
 
 generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-  - files:
-      - controller_manager_config.yaml
-    name: manager-config
+- files:
+  - controller_manager_config.yaml
+  name: manager-config
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
-  - name: controller
-    newName: controller
-    newTag: latest
+- name: controller
+  newName: tibbar/of-operator
+  newTag: v0.0.5.8
```

Comment:
This file changes anytime we deploy with a different image, do we need to .gitignore it?

Reply:
good idea